operations/src/main/java/edu/unc/lib/boxc/operations/api/order/MemberOrderHelper.java

```diff
@@ -46,4 +52,33 @@ public static String formatUnsupportedMessage(PID pid, ResourceType resourceType
         return "Object " + pid.getId() + " of type " + resourceType.name()
                 + " does not support member ordering";
     }
+
+    /**
+     *
+     * @param parentId UUID of parent object
+     * @param reason reason child objects may not be ordered
+     * @param problemPids PIDs of child objects that are not valid to be ordered
+     * @return formatted error message
+     */
+    public static String formatErrorMessage(String parentId, String reason, Collection<PID> problemPids) {
+        return "Invalid request to set order for " + parentId
```

Comment:
I'm trying to decide if it reads weird or not that the REMOVE_FROM operation would emit an error saying that "set order" is not supported. Maybe its fine, since what we're reporting is really that the object type doesn't support ordering at all, rather than that a particular order modification operation isn't supported. What do you think?

Reply:
hmm, i suppose i could change it to something like `Invalid request to amend order for...` ?

Reply:
Hm, that sounds okay but we aren't using "amend" anywhere else related to order. Another could be "Invalid member ordering request for". Passing in the verb might be easier than coming up with verbiage that works for everything though.

Reply:
alright, i just passed it in